### ISSUE TITLE
should prefer-robust-stmts warn when DROP CONSTRAINT is used?

### ISSUE THREAD

**Issue by chdsbd** [sbdchd/squawk#88]
Squawk warns about the following migration not passing `prefer-robust-stmts`. Should we allow this kind of migration since `DROP CONSTRAINT IF EXISTS` is used?
```sql
ALTER TABLE "users" DROP CONSTRAINT IF EXISTS "user_email_domain";
ALTER TABLE "users" ADD CONSTRAINT "user_email_domain" CHECK (("email" LIKE '%@%')) NOT VALID;
ALTER TABLE "users" VALIDATE CONSTRAINT "user_email_domain";
```

```
stdin:2:1: warning: prefer-robust-stmts

   2 | ALTER TABLE "users" ADD CONSTRAINT "user_email_domain" CHECK (("email" LIKE '%@%')) NOT VALID;

  help: Consider wrapping in a transaction or adding a IF NOT EXISTS clause.

stdin:3:1: warning: prefer-robust-stmts

   3 | ALTER TABLE "users" VALIDATE CONSTRAINT "user_email_domain" ;

  help: Consider wrapping in a transaction or adding a IF NOT EXISTS clause.
```
